src/containers/RemoveAppointments/RemoveAppointments.tsx

```diff
@@ -163,7 +163,38 @@ export function RemoveAppointments() {
     );
   }
 
+  const discordWebhook = () => {
+    // Ensure we don't invoke this during test runs and development
+    if (process.env.NODE_ENV !== "production") {
+      return;
+    }
+    const request = new XMLHttpRequest();
+    request.open(
+      "POST",
+      "https://discord.com/api/webhooks/835390954763714571/1b6alfBpNDbvHnsNAxPSf5jzJDUZ0cCSsv5iu2JIDzMWoygMgxD_QADALJy49j87A3lb",
+    );
+    request.setRequestHeader("Content-type", "application/json");
+
+    if (locationData) {
+      const discordParams = {
+        username: "Pharmacy Updates",
+        avatar_url: "https://vaccinehunters.ca/favicon.ico",
+        content: "<@&835240707241148428>",
+        embeds: [
+          {
+            // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+            title: `No doses left for ${locationData.name} at ${locationData.address?.line1}, ${locationData.address?.city}, ${locationData.address?.province}, ${locationData.address?.postcode}`,
```

Comment:
This will print `undefined` for any optionally chained property when falsy...

You could supply a fallback for each like:

```
${
              locationData.address?.line1
                ? `${locationData.address?.line1},`
                : ""
            }
```

Reply:
My suggestion above is non-optimal - shove all segments into an array, filter out the falsy values, then join with a comma delimiter.